backend/push.ts

```
@@ -0,0 +1,66 @@
+      sessionStore.all((err, sessions) => {
+        if (err) {
+          return reject(err)
+        }
+        if (Array.isArray(sessions)) {
+          resolve(sessions)
+        }
+      })
+    })
+    const userSessions = sessions.filter((session) => {
+      return (
+        session.passport && session.passport.user && session.passport.user._id && session.passport.user._id.toString() === userId.toString()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)